content/sensu-go/5.15/guides/use-apikey-feature.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Use the API key feature"
+linkTitle: "Using API keys"
+description: "In this guide, you'll learn how to use Sensu's API key feature for authentication."
+weight: 37
+version: "5.15"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.15:
+    parent: guides
+---
+
+
+- [API key authentication](#api-key-authentication)
+- [Sensuctl management commands](#sensuctl-management-commands)
+
+The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
```

Comment:
```suggestion
The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. The advantages of authenticating with API keys rather than [access tokens](#authentication-quick-start) include:
- **More efficient integration**: Check and handler plugins and other code can integrate with the Sensu API without implementing the logic required to authenticate via the `/auth` API endpoint to periodically refresh the access token.
- **Improved security**: API keys do not require providing a username and password in check or handler definitions.
- **Better admin control**: API keys can be created and revoked without changing the underlying user's password...but keep in mind that API keys will continue to work even if the user's password changes.

API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
```